src/js/angular/import/directives/import-resource-tree.directive.js

```diff
@@ -314,6 +251,9 @@ function importResourceTreeDirective($timeout, ImportContextService) {
                 } else {
                     ImportResourceTreeService.mergeResourceTree($scope.resources, newResources, isUserImport);
                 }
+                $scope.resourceListUtil = $scope.resources.getResourceListUtil();
+                $scope.resourceListUtil.setNameFilter($scope.filterByFileName);
```

Comment:
The `nameFilter` needs to be updated when the filter changes. Update it in the `filterByFileNameChanged` function.